Documentation/data-collection.md

```diff
@@ -991,6 +991,30 @@ data:
     #
     # apiserver_list_watch_request_success_total:rate:sum represents the rate of change for successful LIST and WATCH requests over a 5 minute period.
     - '{__name__="apiserver_list_watch_request_success_total:rate:sum", verb=~"LIST|WATCH"}'
+    #
+    # owners: (https://github.com/stackrox/stackrox, @stackrox/eng)
+    #
+    # rhacs:telemetry:rox_central_secured_clusters provides the number of clusters secured by a Central instance of a
+    # Red Hat Advanced Cluster Security installation.
+    - '{__name__="rhacs:telemetry:rox_central_secured_clusters"}'
+    #
+    # owners: (https://github.com/stackrox/stackrox, @stackrox/eng)
+    #
+    # rhacs:telemetry:rox_central_secured_nodes provides the number of nodes secured by a Central instance of a
+    # Red Hat Advanced Cluster Security installation.
+    - '{__name__="rhacs:telemetry:rox_central_secured_nodes"}'
+    #
+    # owners: (https://github.com/stackrox/stackrox, @stackrox/eng)
+    #
+    # rhacs:telemetry:rox_central_secured_vcpu provides the number of vCPUs secured by a Central instance of a
+    # Red Hat Advanced Cluster Security installation.
+    - '{__name__="rhacs:telemetry:rox_central_secured_vcpu"}'
+    #
+    # owners: (https://github.com/stackrox/stackrox, @stackrox/eng)
+    #
+    # rhacs:telemetry:rox_sensor_info provides information about a Sensor instance of a Red Hat Advanced
+    # Cluster Security installation.
+    - '{__name__="rhacs:telemetry:rox_sensor_info"}'
```

Comment:
Again looking at the recording expression:

```
max by (build, central_id, hosting, install_method, sensor_id, sensor_version) (
  rox_sensor_secured_nodes{branding=~"rhacs|RHACS"} > bool 0
)
```

Not sure why "> bool 0" is needed. The assumption for an info metric is that it always reports 1.

```
group by (build, central_id, hosting, install_method, sensor_id, sensor_version) (
  rox_sensor_secured_nodes{branding=~"rhacs|RHACS"}
)
```